libquickevent/libquickeventgui/src/reportoptionsdialog.cpp

```diff
@@ -123,13 +146,33 @@ bool ReportOptionsDialog::isStartListPrintStartNumbers() const
 	return ui->chkStartOpts_PrintStartNumbers->isChecked();
 }
 
-QString ReportOptionsDialog::sqlWhereExpression() const
+QString ReportOptionsDialog::sqlWhereExpression(const int stage_id) const
 {
 	const Options opts = options();
-	return sqlWhereExpression(opts);
+	return sqlWhereExpression(opts,stage_id);
 }
 
-QString ReportOptionsDialog::sqlWhereExpression(const ReportOptionsDialog::Options &opts)
+QString ReportOptionsDialog::getClassesForStartNumber(const int number, const int stage_id)
```

Comment:
I think, that we should rather add `startNUmber` or `startName` or `startId` column to classes.

Reply:
Start number is used only in 2 places. 
- Startlist print (for classes & starters)
- XML Startlist export 

I didn't find it necessary to add a column to the db for this.

Reply:
If we plan to start to support more starts, then we should also give user some UI to work with. Use start-code number for this is cryptic and hard to find for user. So I suggest to either support more starts in correct way or nothing.

What user will certainly want is:
1. possibility to define more starts, `courses` table is best place IMO.
2. Possibility to give start some name.

What we need to have this:
1. add one column to `courses` table
2. define one column in UI table

This is IMO more straightforward solution than mangle start-name to code of start station in codes. Also users will be happy, because they will got table column, where they can put start-name/number, whatever they want, on the place where they will be looking for such kind of information.

Reply:
Its for discusion. Maybe from people who wrote to affected issues.
For me :
 - Start codes/names are imported from courses. I've never didn't need to change it. I only use it from courses. 
 - And I only need it to print startlist for starters.
 - I don't need to edit or manage it in any way.

Reply:
I wrote this because I need it for WMOC.

Reply:
Ok, let to it this way for now. I realized that, if we introduce new DB column, then we can have different start-number defined in `courses` table and different start-code in `classcodes`. This is something also not wanted.